applications/fishing-map/src/features/dataviews/dataviews.slice.ts

```diff
@@ -6,13 +6,18 @@ import { AsyncReducer, createAsyncSlice } from 'utils/async-slice'
 import { RootState } from 'store'
 import { getDatasetByDataview } from 'features/workspace/workspace.slice'
 import { fetchDatasetsByIdsThunk } from 'features/datasets/datasets.slice'
+import mockedDataviews from './dataviews.mock'
```

Comment:
I'm not sure having a mock file around that is only for "personal" use is a good idea. How about we have a `./dataviews.mock.sample` in git, but the actual file is not, and import the actual file when an env var is set to true?